sema/commons/web/parse_headers.py

```
@@ -0,0 +1,35 @@
+    msg = EmailMessage()
+    mode = mode.lower()
+    msg[mode] = content
+    if mode == "content-type":
+        main = msg.get_content_type()
+    elif mode == "content-disposition":
+        main = msg.get_content_disposition()
+    else:
+        raise ValueError(
+            "mode should be one of 'content-type' (default) "
+            "or 'content-disposition'"
+        )
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)